setup.py

```diff
@@ -117,6 +116,11 @@ def _read_file(filename):
         'zope.traversing',
         'zope.viewlet',
     ],
+    extras_require={
+        ':python_version=="2.7"': [
```

Comment:
Does it work that way with the colon `:` in front? The [documentation](https://setuptools.readthedocs.io/en/latest/setuptools.html#declaring-platform-specific-dependencies) seems a bit unclear to me.

Reply:
It tried it with `buildout` and it worked: It was installed on Python 2.7 but not on Python 3.

Reply:
I have taken the approach from https://github.com/inveniosoftware/troubleshooting/issues/1

Reply:
It's also possible to do this directly on a requirement in `install_requires`:

```python
install_requires=[
    'ipaddress ; python_version == "2.7"',
]
```

(That may require a newer setuptools.)